lib/tldr/argv_parser.rb

```diff
@@ -78,6 +78,10 @@ def parse args, options = {cli_defaults: true}
           options[:verbose] = verbose
         end
 
+        opts.on CONFLAGS[:verbose_cancelled_trace], "Print stack traces for cancelled tests" do |verbose_cancelled_trace|
+          options[:verbose_cancelled_trace] = verbose_cancelled_trace
+        end
+
```

Comment:
I don't like this name, because I don't think of tests that were interrupted due to the timeout as being "cancelled" (I think a Ctrl-C interrupt would be someone "canceling" them). `--print-interrupted-test-backtraces`? I don't know a better name for this. @tenderlove?

Reply:
I totally agree with your reasoning – `--print-interrupted-test-backtraces` sounds great to me, but I'll wait to hear more suggestions though =)

Reply:
Renamed in 8d50b94 =)